# Our Favorite Papers of 2015

There were a lot of really cool advances in our understanding energy balance and the signalling mechanisms that underlie endocrine control of appetite and metabolism.  Here are a few of the papers that we thought were the most interesting from the year.

## Regulation of Energy Balance

* Tsai S, Sitzmann JM, Dastidar SG, Rodriguez A a, Vu SL, McDonald CE, Academia EC, O’Leary MN, Ashe TD, La Spada AR, Kennedy BK. Muscle-specific 4E-BP1 signaling activation improves metabolic parameters during aging and obesity. J Clin Invest 125: 2952–64, 2015. doi: [10.1172/JCI77361](http://dx.doi.org/10.1172/JCI77361).
* Guridi M, Tintignac LA, Lin S, Kupr B, Castets P, Rüegg MA. Activation of mTORC1 in skeletal muscle regulates whole-body metabolism through FGF21. Sci Signal 8: ra113–ra113, 2015. doi: [10.1126/scisignal.aab3715](http://dx.doi.org/10.1126/scisignal.aab3715).

4EBP1 is a translational repressor that is inhibited by mTORC1 signaling.  [Brian Kennedy's group at the Buck Institute](http://www.buckinstitute.org/kennedyLab) described a constitutively active 4EBP1 transgenic and studied the effects of its over-expression in both fat and muscle tissue.  This was followed up shortly by [Markus Rüegg's group at Universitat Basel](http://www.biozentrum.unibas.ch/research/groups-platforms/overview/unit/rueegg/), who looked at constitutively active mTORC1 muscles (via *Tsc1* knockout).  Interestingly in both cases muscle mice had a fiber-type switch (towards oxidative fibers), increased energy expenditure and reduced adipose tissue.  Both papers also implicate muscle derived FGF21 as medating some of these effects.  Both papers nicely highlighted the tissue-specific roles of mTORC1 in energy homeostasis.

* Han J, Li E, Chen L, Zhang Y, Wei F, Liu J, Deng H, Wang Y. The CREB coactivator CRTC2 controls hepatic lipid metabolism by regulating SREBP1. Nature 2015. doi: [10.1038/nature14557](http://dx.doi.org/10.1038/nature14557).

The way in which mTORC1 activates SREBP1c to promote lipid synthesis has proven elusive for quite some time.  This paper from [Yighuo Wang's group at Tsinghua University](http://life.tsinghua.edu.cn/english/faculty/faculty/2099.html) suggests that at least part of this is through phosphorylation of CRTC2 by mTORC1 which interrupts the trafficking and processing of SREBP1c in the Golgi.

* Koch M, Varela L, Kim JG, Kim JD, Hernández-Nuño F, Simonds SE, Castorena CM, Vianna CR, Elmquist JK, Morozov YM, Rakic P, Bechmann I, Cowley M A., Szigeti-Buck K, Dietrich MO, Gao X-B, Diano S, Horvath TL. Hypothalamic POMC neurons promote cannabinoid-induced feeding. Nature 2015. doi: [10.1038/nature14260](http://dx.doi.org/10.1038/nature14260).

Endogenous and exogenous cannabinoids promote the motivation to eat.  In this paper, from [Tamas Horvath's group at Yale](https://medicine.yale.edu/lab/horvath/), they show by [DREAD](https://en.wikipedia.org/wiki/Receptor_activated_solely_by_a_synthetic_ligand)-mediated inhbition that POMC neurons are required for cannabinoid-induced feeing.  They also show that there is selective secretion of beta-endorphin, but not alpha-MSH in these neurons, suggesting a novel secretory mechanism for these [ACTH](https://en.wikipedia.org/wiki/Receptor_activated_solely_by_a_synthetic_ligand)-derived neuropeptides

* Morley TS, Xia JY, Scherer PE. Selective enhancement of insulin sensitivity in the mature adipocyte is sufficient for systemic metabolic improvements. Nat Commun 6: 7906, 2015. doi: [10.1038/ncomms8906](http://dx.doi.org/10.1038/ncomms8906)

We normally think of adipocyte hypertrophy as being a bad thing, as elevations in fat mass are often co-morbid with metabolic syndrome.  This paper from [Philip Scherer's group at UTSW](http://www4.utsouthwestern.edu/schererlab/) shows that by enhancing adipose insulin sensitivity (via inducible deletion of PTEN) can improve overall insulin-sensitivity in spite of elevated fat storage.  Even more interestingly, adipose inflammation is also reduced, separating adipocyte hypertrophy from macrophage infiltration.  

## Large Scale Datasets and Functional Analyses

* Rivas MA, Pirinen M, Conrad DF, Lek M, Tsang EK, Karczewski KJ, Maller JB, Kukurba KR, DeLuca DS, Fromer M, Ferreira PG, Smith KS, Zhang R, Zhao F, Banks E, Poplin R, Ruderfer DM, Purcell SM, Tukiainen T, Minikel E V, Stenson PD, Cooper DN, Huang KH, Sullivan TJ, Nedzel J, Bustamante CD, Li JB, Daly MJ, Guigo R, Donnelly P, Ardlie K, Sammeth M, Dermitzakis ET, McCarthy MI, Montgomery SB, Lappalainen T, MacArthur DG, Segre A V., Young TR, Gelfand ET, Trowbridge CA, Ward LD, Kheradpour P, Iriarte B, Meng Y, Palmer CD, Esko T, Winckler W, Hirschhorn J, Kellis M, Getz G, Shablin AA, Li G, Zhou Y-H, Nobel AB, Rusyn I, Wright FA, Battle A, Mostafavi S, Mele M, Reverter F, Goldmann J, Koller D, Gamazon ER, Im HK, Konkashbaev A, Nicolae DL, Cox NJ, Flutre T, Wen X, Stephens M, Pritchard JK, Tu Z, Zhang B, Huang T, Long Q, Lin L, Yang J, Zhu J, Liu J, Brown A, Mestichelli B, Tidwell D, Lo E, Salvatore M, Shad S, Thomas JA, Lonsdale JT, Choi RC, Karasik E, Ramsey K, Moser MT, Foster BA, Gillard BM, Syron J, Fleming J, Magazine H, Hasz R, Walters GD, Bridge JP, Miklos M, Sullivan S, Barker LK, Traino H, Mosavel M, Siminoff LA, Valley DR, Rohrer DC, Jewel S, Branton P, Sobin LH, Barcus M, Qi L, Hariharan P, Wu S, Tabor D, Shive C, Smith AM, Buia SA, Undale AH, Robinson KL, Roche N, Valentino KM, Britton A, Burges R, Bradbury D, Hambright KW, Seleski J, Korzeniewski GE, Erickson K, Marcus Y, Tejada J, Taherian M, Lu C, Robles BE, Basile M, Mash DC, Volpi S, Struewing JP, Temple GF, Boyer J, Colantuoni D, Little R, Koester S, Carithers LJ, Moore HM, Guan P, Compton C, Sawyer SJ, Demchok JP, Vaught JB, Rabiner CA, Lockhart NC, Friedlander MR, ’t Hoen PAC, Monlong J, Gonzalez-Porta M, Kurbatova N, Griebel T, Barann M, Wieland T, Greger L, van Iterson M, Almlof J, Ribeca P, Pulyakhina I, Esser D, Giger T, Tikhonov A, Sultan M, Bertier G, Lizano E, Buermans HPJ, Padioleau I, Schwarzmayr T, Karlberg O, Ongen H, Kilpinen H, Beltran S, Gut M, Kahlem K, Amstislavskiy V, Stegle O, Flicek P, Strom TM, Lehrach H, Schreiber S, Sudbrak R, Carracedo A, Antonarakis SE, Hasler R, Syvanen A-C, van Ommen G-J, Brazma A, Meitinger T, Rosenstiel P, Gut IG, Estivill X. Effect of predicted protein-truncating genetic variants on the human transcriptome. Science 348: 666–669, 2015. doi: [10.1126/science.1261877](http:/dx.doi.org/10.1126/science.1261877).

* Ardlie KG, Deluca DS, Segre A V., Sullivan TJ, Young TR, Gelfand ET, Trowbridge CA, Maller JB, Tukiainen T, Lek M, Ward LD, Kheradpour P, Iriarte B, Meng Y, Palmer CD, Esko T, Winckler W, Hirschhorn JN, Kellis M, MacArthur DG, Getz G, Shabalin AA, Li G, Zhou Y-H, Nobel AB, Rusyn I, Wright FA, Lappalainen T, Ferreira PG, Ongen H, Rivas MA, Battle A, Mostafavi S, Monlong J, Sammeth M, Mele M, Reverter F, Goldmann JM, Koller D, Guigo R, McCarthy MI, Dermitzakis ET, Gamazon ER, Im HK, Konkashbaev A, Nicolae DL, Cox NJ, Flutre T, Wen X, Stephens M, Pritchard JK, Tu Z, Zhang B, Huang T, Long Q, Lin L, Yang J, Zhu J, Liu J, Brown A, Mestichelli B, Tidwell D, Lo E, Salvatore M, Shad S, Thomas JA, Lonsdale JT, Moser MT, Gillard BM, Karasik E, Ramsey K, Choi C, Foster BA, Syron J, Fleming J, Magazine H, Hasz R, Walters GD, Bridge JP, Miklos M, Sullivan S, Barker LK, Traino HM, Mosavel M, Siminoff LA, Valley DR, Rohrer DC, Jewell SD, Branton PA, Sobin LH, Barcus M, Qi L, McLean J, Hariharan P, Um KS, Wu S, Tabor D, Shive C, Smith AM, Buia SA, Undale AH, Robinson KL, Roche N, Valentino KM, Britton A, Burges R, Bradbury D, Hambright KW, Seleski J, Korzeniewski GE, Erickson K, Marcus Y, Tejada J, Taherian M, Lu C, Basile M, Mash DC, Volpi S, Struewing JP, Temple GF, Boyer J, Colantuoni D, Little R, Koester S, Carithers LJ, Moore HM, Guan P, Compton C, Sawyer SJ, Demchok JP, Vaught JB, Rabiner CA, Lockhart NC, Ardlie KG, Getz G, Wright FA, Kellis M, Volpi S, Dermitzakis ET. The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans. Science 348: 648–660, 2015. doi: [10.1126/science.1262110](http://dx.doi.org/10.1126/science.1262110).

## Reviews

* Hawley JA, Maughan RJ, Hargreaves M. Exercise Metabolism: Historical Perspective. Cell Metab 22: 12–17, 2015. doi: [10.1016/j.cmet.2015.06.016](http://dx.doi.org/10.1016/j.cmet.2015.06.016).

* Efeyan A, Comb WC, Sabatini DM. Nutrient-sensing mechanisms and pathways. Nature 517: 302–310, 2015. doi: [10.1038/nature14190](http://dx.doi.org/10.1038/nature14190).
